apps/quiz/models.py

```diff
@@ -34,6 +34,7 @@ class Quiz(db.Model):
 
     description = db.Column(db.String(500))
     questions = db.relationship('Question', backref='quiz', lazy=True)
+    author = db.relationship('User')
```

Comment:
I  see no reason for this field here, if that's an old field (some leftover) it should be removed

Reply:
There is username under each quiz, thanks to this relationship I can get to it like this `quiz.author.username`. Is there better way?

Reply:
Sure but what you are doing here, is clearly repeating the code, take a look at main branch:
- there already exists the field author_id so there is a relation One to Many, there is  no need to define another relation
- You can achieve this thing you are talking about using @property, simply use author_id to get author instance and return this object in property called author, if